### ISSUE TITLE
InMobi adapter bug of notify recyclerView refresh data

### ISSUE THREAD

**Issue by xavierwangg** [googleads/googleads-mobile-android-mediation#161]
InMobi native can be showed perfectly after inflated the first time. But when change the native ad position, or just call notifyDataSetChange() without any change in original list data. The InMobi native ad doesn't show mediaView(image or video) while icon, title text, body text can be showed good; and all ad elements can't be clicked. But AdMob native will be OK after refresh.

I tried to use 'com.google.ads.mediation:inmobi-imageview:7.2.2', this library can work good.
So it just the problem of 'com.google.ads.mediation:inmobi:7.2.7.0'. 

Hope you can fix this bug.

You can see the screenshot here.
[https://i.stack.imgur.com/wTVFg.png](url)

**Comment by spilaka** [comment]
@BlackComte  facing the same issue with latest InMobi Android Adapter ?

**Comment by awayo** [comment]
Yes, only com.google.ads.mediation:inmobi-imageview adapters works as expected.

**Comment by RajSuvariya-InMobi** [comment]
@BlackComte @awayo The latest adapter 9.0.6.0 has the fixes for the above-mentioned issues. Please test and let us know if it still exists.
